pkg/execute/kubectl_cmd_builder.go

```diff
@@ -12,19 +12,23 @@ import (
 	"github.com/slack-go/slack"
```

Comment:
We could hide `auth` command as it doesn't seem to be useful without flags:
<img width="491" alt="Screenshot 2022-10-12 at 09 32 58" src="https://user-images.githubusercontent.com/7155799/195278873-b01980c2-23a3-4398-980b-e064a59fe024.png">

WDYT?

Reply:
the `auth` was already removed from the default configuration. I also removed the `diff` as it also doesn't make sens. 
Not sure why you mentioned flags though 🤔  you can run it like `kubectl auth can-i get pods`. However, this is sth which is not true, as it's not going through our "policy engine" and as Botkube has cluster admin, it can do everything but we still have restrictions per channel.



As you can see, I introduced a Guard which I called dummy. I thought that the new Guard will handle such cases. If not, I can create a wrapper around the real Guard.

Reply:
I meant hiding the option to execute `auth` from interactive kubectl, as this result in the response I posted.
![Screenshot 2022-10-12 at 10 23 04](https://user-images.githubusercontent.com/7155799/195290267-563c9723-dfd3-42ce-87b0-60c913f94bf8.png)

We agreed Guard has two methods, and neither of them returns a list of verbs, both of them work with a selected verb:
```go
GetAllowedResourcesForVerb(verb string, allConfiguredResources []string) ([]Resource, error)
GetResourceDetails(selectedVerb, resourceType string) (Resource, error)
```

So displaying verbs is something before using guard. But we can add another method for it, which will simply filter not  supported commands by the interactive kubectl. I can do that 👍 Just wanted to hear your thoughts about hiding this option even if it is configured for executors.